### PR TITLE
Added clear button to search bar

### LINKED ISSUE #180: Add search bar clear button
Add a small x to the search bar when it has some text to reset the text. Like this:

image

### PR DESCRIPTION
**Pull request by RunDevelopment**
Fixes #180.

No search query:

image

With search query:

image

### CODE REVIEW

File: src/global.css [hunk 1]
```diff
@@ -91,3 +91,14 @@ body {
...
+.clear-search-bar-button {
+    color: var(--chakra-colors-gray-300);
+    cursor: pointer;
+}
+.clear-search-bar-button:hover {
+    color: var(--chakra-colors-gray-000);
+}
+.clear-search-bar-button > * {
+    font-size: 66%;
+}
```
Comment (joeyballentine) [inline src/global.css:104]: How does this look in light theme?

Comment (joeyballentine) [reply src/global.css:104]: And why regular css instead of chakra props?

Comment (joeyballentine) [reply src/global.css:104]: Also the X should probably be darker in light mode, its kinda hard to see right now

Comment (RunDevelopment) [reply src/global.css:104]: In that case, we should probably adjust the chakra variables because the magnifying glass isn't any better.

Comment (joeyballentine) [reply src/global.css:104]: The chakra variables you used are just colors. They don't change between dark and light. That's the reason I use useColorModeValue everywhere. If you really wanna do it in regular css, do it like how Zamboni did the controls UI where it selects light and dark theme and use the different color variables. But it's probably easier just to use the style props

Comment (RunDevelopment) [reply src/global.css:104]: Done.

image

**Pull request merged**